src/LanguageServer/Protocol/Handler/References/FindImplementationsHandler.cs

```diff
@@ -33,26 +34,32 @@ public FindImplementationsHandler(IGlobalOptionService globalOptions)
 
         public LSP.TextDocumentIdentifier GetTextDocumentIdentifier(LSP.TextDocumentPositionParams request) => request.TextDocument;
 
-        public async Task<LSP.Location[]> HandleRequestAsync(LSP.TextDocumentPositionParams request, RequestContext context, CancellationToken cancellationToken)
+        public Task<LSP.Location[]> HandleRequestAsync(LSP.TextDocumentPositionParams request, RequestContext context, CancellationToken cancellationToken)
```

Comment:
assuming that you dropped `async` is intentional here

Reply:
Indeed it would be a compiler warning if I left it in place.

If an async method only has a single `await` at the end, it is usually better to remove it, and the `async` modifier, and just return the `Task` directly. This avoids the creation of a state machine and so gives a slight perf bump. It does however change the call stack for exceptions that might be thrown so I wouldn't say its a hard and fast rule.